### DISCUSSION TITLE
Adding loading Indicator

### DISCUSSION THREAD

**Discussion by ManojKiranA** [filamentphp/filament#1331] [Ideas]
Add the Loading Indicator in Following Areas of Filament.

- [x] 1. Global Search
- [x] 2. Table Level Search
- [x] 3. Buttons in Forms
- [x] 4. Pagination Limit change
- [x] 5. Selecting models while bulk selecting
- [x] 6. Filtering Records
- [x] 7. Buttons on Relation Modals

**Comment by michaelgatuma** [comment]
This would be nice because the user wonders why nothing is happening during the livewire requests round trip, worse when internet is slow. I had to do `wire:loading.class="opacity-50"` around the table. Not the best method but its better than nothing.

**Comment by danharrin** [comment]
All buttons now have a loading state, which is a good first step.

**Reply by ManojKiranA** [reply]
@danharrin  Can i close this I think loading indicator is added to most of the components
